content/en/about/ecosystem/index.md

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
Can we add a todo on this page as well

Reply:
Done.
Added an extra one in the `data/companies.yml` because it also contains a list of placeholder distributors, integrations and pro services.